### PR TITLE
Philips.HueSync version 1.8.2.31

### PR DESCRIPTION
**Pull request by TheJanzap**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80699)

### CODE REVIEW

File: manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Philips.HueSync
+PackageVersion: 1.8.2.31
+MinimumOSVersion: 10.0.0.0
+InstallerSwitches:
+  Silent: /S /v/qn
+  SilentWithProgress: /S /v/qn
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml:11]:
```suggestion
- Architecture: x64
```

File: manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://firmware.meethue.com/storage/huesyncwin/31/0d9249b9-38f6-4eb0-9046-c1bc8ea3d767/HueSyncInstaller_1.8.2.31.exe
+  InstallerSha256: D83C3681FC1CF7EEF8D9FA4F1F4B797E354A2D6A571937B213DDF03B625ACB1C
+  AppsAndFeaturesEntries:
+  - DisplayName: Hue Sync
+    Publisher: Signify Netherlands B.V.
+    ProductCode: '{C0270355-35E2-4862-8B57-A7C1A258AF77}'
```
Comment (Trenly) [inline manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml:19]:
```suggestion
    ProductCode: '{E971B544-FF86-48F0-BA48-8491FF25D75F'

```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://firmware.meethue.com/storage/huesyncwin/31/0d9249b9-38f6-4eb0-9046-c1bc8ea3d767/HueSyncInstaller_1.8.2.31.exe
+  InstallerSha256: D83C3681FC1CF7EEF8D9FA4F1F4B797E354A2D6A571937B213DDF03B625ACB1C
+  AppsAndFeaturesEntries:
+  - DisplayName: Hue Sync
+    Publisher: Signify Netherlands B.V.
+    ProductCode: '{E971B544-FF86-48F0-BA48-8491FF25D75F'
+
```
Comment (ItzLevvie) [inline manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml:20]:
```suggestion
    ProductCode: '{E971B544-FF86-48F0-BA48-8491FF25D75F}'
```

Comment (ItzLevvie) [reply manifests/p/Philips/HueSync/1.8.2.31/Philips.HueSync.installer.yaml:20]: Looks like there was a missing `}`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**